### DISCUSSION TITLE
Parameters for GitHub Actions Certification exam from GitHub Partner Certifications

### DISCUSSION THREAD

**Discussion by michael-angelok** [community/community#58613] [Actions] [answered]
Hi, I am practicing for this GitHub Actions cert and want to know the parameters of this exam.

How long is the exam?

How many questions will there be?

Any resources or services that enables you to practice questions for this exam? etc.

Would appreciate any info and/or tips on this :)

**Comment by michael-angelok** [comment]
I can confirm that there 75 question, and lasts 2 hours. Although I still haven't found other resource material

**Comment by Akash1134** [comment, marked as answer]
Hi @Michael-AngeloK ,

Please check out the [FAQ's for the GitHub Partner certifications](https://examregistration.github.com/faq).  

You may also see the [Candidate Handbook UAT](https://examregistration.github.com/handbook)

> Although I still haven't found other resource material

GitHub offers a variety of self-paced learning content on [Microsoft Learn](https://learn.microsoft.com/en-us/users/githubtraining/collections/n5p4a5z7keznp5). This content helps build necessary technical skills to advance your knowledge and ability to address solutions. These browser-based trainings are available to complete at your own pace.

[GitHub Actions Docs](https://docs.github.com/en/actions/learn-github-actions) is another comprehensive resource to help you learn and prepare.

If you have further questions, please feel free to post in the Community Discussions
